plugins/omar-wms-plugin/grails-app/services/omar/wms/WebMappingService.groovy

```diff
@@ -54,6 +54,16 @@ class WebMappingService implements InitializingBean
     def docTypeLocation = grailsLinkGenerator.link( absolute: true, uri: "/schemas/wms/1.1.1/WMS_MS_Capabilities.dtd" )
     def model = geoscriptService.capabilitiesData
 
+    def otherParams = [startTime: new Date(), internalTime: new Date(), procTime: new Date()]
```

Comment:
This 'otherParams' is confusing. The intent later in the code gets lost. I would explicitly define each of these variables.

Reply:
turns out, "otherParams" is used in a lot of the methods and removing it breaks WMS. Not "this" otherParams, but one that already existed..